src/cmo.adoc

```diff
@@ -385,8 +385,8 @@ translation, the instruction does _not_ check the accessed and dirty bits and
 neither raises an exception nor sets the bits.
 
 When a page fault, guest-page fault, or access fault exception is taken, the
-relevant *tval CSR is written with the faulting effective address (i.e. the same
-faulting address value as for other causes of these exceptions).
+relevant *tval CSR is written with the faulting effective address (i.e. the
+value of _rs1_).
```

Comment:
This is the part that I endorse.  My initial reaction is the other changes are not necessary, and maybe not even desirable, but I'll leave that decision to @dkruckemyer-ventana.

Reply:
I think at least the bits about not requiring `rs1` to be aligned are needed because that isn't specified anywhere currently (unless I missed it).

Reply:
I did not think there was an intent to support a CMO for flush/invalidate/clean to operate on multiple cache blocks. Its not clearly documented but it is more reasonable to think of a CMO.flush/invalidate/clean as having an access width of 1 i.e. it targets the cache block addressed by `rs1` and that cache block may be addressed by specifying any byte of that cache block.

Reply:
I agree, that's what I meant it to say. Does it sound like it's saying it would operate on multiple cache blocks?